### DISCUSSION TITLE
Questions?

### DISCUSSION THREAD

**Discussion by CanPixel** [CanPixel/BadOptics#87] [Q&A]
Questions about the game, or more general questions for me are welcome

**Comment by ThatOneGuy301** [comment]
Is it possible to win the game?

**Reply by Fengsemul** [reply]
In https://www.reddit.com/r/BadOptics/comments/w0zgb5/does_anyone_want_a_mod_or_something_to_stop_the/ , he replys with "no worries, imma fix it!". He may indirectly admit that ["win by diplomatic ways"](https://archive.ph/o/RM2Aj/https://www.reddit.com/r/BadOptics/comments/w0zgb5/does_anyone_want_a_mod_or_something_to_stop_the/) counts as such.

Conclusion:
So in canon, "Geopolitical Stability" may implies that the country in the creator's imagination continues and counts as an approval "win" in to [receive](https://dictionary.cambridge.org/dictionary/english/receive) something [positive](https://dictionary.cambridge.org/dictionary/english/positive)  ("World Peace!") from game as a whole ending, such as [approval](https://dictionary.cambridge.org/dictionary/english/approval) from game maker due maybe exciting "!" in  "World Peace!", because the player had [earned](https://dictionary.cambridge.org/dictionary/english/earn) it.

Comments about winning condition:
From https://docs.google.com/forms/d/e/1FAIpQLSfaYvM_MxY94tz7vUZXI46aOBoGzG7wufuw9nIi5XW3VMQTtw/viewanalytics

> The "win condition" feels really arbitrary and unconnected to the gameplay, making it feel harder would make it feel more important

Even his Co-Tailor & friend Jreg as the officially posted video https://www.youtube.com/watch?v=fEP3858pZuM in https://canpixel.com/BadOptics/ said: 


0:07

> imagine a world of pure psychic energy

0:10

> in which ideas battle it out for

0:12

> supremacy you control three feuding

0:14

> ideologies all of whom have banded

0:16

> together in order to survive in this

0:18

> world

1:15: 

> win by war win by

1:17:

> diplomacy

Defining Winning condition:
The game ignores being the only govering body running so https://www.reddit.com/r/BadOptics/comments/vu2xs4/theres_noone_left_did_i_just_win/ falls off and the first definition due no price gaining from game itself and no timeline end. Also a competition called [situation](https://dictionary.cambridge.org/dictionary/english/situation) in which someone is [trying](https://dictionary.cambridge.org/dictionary/english/trying) to [win](https://dictionary.cambridge.org/dictionary/english/win) something or be more [successful](https://dictionary.cambridge.org/dictionary/english/successful) than someone [else](https://dictionary.cambridge.org/dictionary/english/else) is blank to https://canpixel.com/BadOptics/v1/index.html 
win
verb
uk
/wɪn/ us
/wɪn/
present participle winning | past tense and past participle won
A2 [[ I or T ]](https://dictionary.cambridge.org/help/codes.html)
to [achieve](https://dictionary.cambridge.org/dictionary/english/achieve) first [position](https://dictionary.cambridge.org/dictionary/english/position) and/or get a [prize](https://dictionary.cambridge.org/dictionary/english/prize) in a [competition](https://dictionary.cambridge.org/dictionary/english/competition), [election](https://dictionary.cambridge.org/dictionary/english/election), [fight](https://dictionary.cambridge.org/dictionary/english/fight), etc.
C2 [[ T ]](https://dictionary.cambridge.org/help/codes.html)
to [receive](https://dictionary.cambridge.org/dictionary/english/receive) something [positive](https://dictionary.cambridge.org/dictionary/english/positive), such as [approval](https://dictionary.cambridge.org/dictionary/english/approval), [loyalty](https://dictionary.cambridge.org/dictionary/english/loyal), or [love](https://dictionary.cambridge.org/dictionary/english/love) because you have [earned](https://dictionary.cambridge.org/dictionary/english/earn) it
win
verb [[ I/T ]](https://dictionary.cambridge.org/help/codes.html)
us
/wɪn/
present participle winning | past tense and past participle won us/wʌn/
to [defeat](https://dictionary.cambridge.org/dictionary/english/defeat) a [competitor](https://dictionary.cambridge.org/dictionary/english/competition), or to [achieve](https://dictionary.cambridge.org/dictionary/english/achieve) first [position](https://dictionary.cambridge.org/dictionary/english/position) or get a [prize](https://dictionary.cambridge.org/dictionary/english/prize) in a [competition](https://dictionary.cambridge.org/dictionary/english/competition)

There's symbolic disapproval by losing war and being overthrown.

Creator's mind:
https://canpixel.com/index.php:

> It is important to have clear visions of some (unrelated) things you want before you begin.
> Nihilism indirectly seeps into design, if you're not motivated enough, the game might reflect that.
> Treat the game as an extension of you, be the creator and the creation.
> Opening room for being passionate about your concept, being able to romanticize it fully makes for authenticity.
> 
> Not unlike method acting, I face my work as a "method developer", artistically.
> For periods I breathe in abstract, I let it be me. Me being the vessel for the collected ideas, eventually converging them into one greater whole that I can then sketch out and realize.
> This captures the necessary passion (and other human aspects) I use to let the creation speak for itself, ontologically involving me with the creation. At the very least being a profound reflection of my life at that time.
> 
> This way I can ensure that I'm able to bring my creative visions to life passionately.
> It's never just a game meant for mass consumption, with one goal. 

Assuming that the creator is motivated, there's a purpose beyond nihilism related to his personality. It has many goals instead like education in https://www.researchgate.net/publication/288235759_Gamification_of_politicsStart_a_new_game and or having many winning goals as the consumer defines. 

Game's and developer's suggestion of winning with biases:
The Tutorial suggests to click on your Ideology-Ball. And has a bias by mentioning to answer on the populations favor, but than refuse it with less confidence. [Gam-Comptutorial-Newspaper-Trytoanswerornot2022-12-11-145026.png](https://postimg.cc/N58hrTrp) 
![GamComptutorialNewspaperTrytoanswerornot2022-12-11 145026](https://user-images.githubusercontent.com/118663590/206907692-8f8bc832-52c9-4137-83ab-45f3e18de361.png)

https://i.postimg.cc/sD2zvZLH/Gam-Comp-Last-Tutorial-Newspaper2022-12-11-135941.png
![GamCompLastTutorialNewspaper2022-12-11 135941](https://user-images.githubusercontent.com/118663590/206905118-c4b28eb0-70da-40c8-b563-a73f68f74c1d.png)
implies that the player should find a way how to run their country.
[CanPixel](https://github.com/CanPixel)CanPixel
[Feb 10](https://github.com/CanPixel/BadOptics/discussions/91#discussioncomment-2151076)
Owner

> Governing a country is hard, there is no one way.
> Go figure it out!
in https://canpixel.com/BadOptics/ comments

governing
adjective [[ before noun ]](https://dictionary.cambridge.org/help/codes.html)
uk
/ˈɡʌv.ən.ɪŋ/ us
/ˈɡʌv.ɚ.nɪŋ/
governing adjective [before noun] (RULING)
having the [power](https://dictionary.cambridge.org/dictionary/english/power) to [govern](https://dictionary.cambridge.org/dictionary/english/govern) a [country](https://dictionary.cambridge.org/dictionary/english/country) or an [organization](https://dictionary.cambridge.org/dictionary/english/organization)
governing adjective [before noun] (INFLUENCING)
having a [controlling](https://dictionary.cambridge.org/dictionary/english/controlling) [influence](https://dictionary.cambridge.org/dictionary/english/influence) on something:
governing
adjective [[ before noun ]](https://dictionary.cambridge.org/help/codes.html)
  GOVERNMENT
uk
/ˈɡʌvənɪŋ/ us
the [management](https://dictionary.cambridge.org/dictionary/english/management) or [control](https://dictionary.cambridge.org/dictionary/english/control) of the [activities](https://dictionary.cambridge.org/dictionary/english/activity) of a [particular](https://dictionary.cambridge.org/dictionary/english/particular) [country](https://dictionary.cambridge.org/dictionary/english/country), [region](https://dictionary.cambridge.org/dictionary/english/region), or [organization](https://dictionary.cambridge.org/dictionary/english/organization)

running
adjective
uk
/ˈrʌnɪŋ/ us
[[ before noun ]](https://dictionary.cambridge.org/help/codes.html)
[happening](https://dictionary.cambridge.org/dictionary/english/happening) [continuously](https://dictionary.cambridge.org/dictionary/english/continuously) without [interruption](https://dictionary.cambridge.org/dictionary/english/interruption)

Combining the first adjective definition from "running" and "governing":
Summary: "having the power to [control](https://dictionary.cambridge.org/dictionary/english/control) and [direct](https://dictionary.cambridge.org/dictionary/english/direct) the [public](https://dictionary.cambridge.org/dictionary/english/public) [activity](https://dictionary.cambridge.org/dictionary/english/activity) of [buying](https://dictionary.cambridge.org/dictionary/english/buy) and [selling](https://dictionary.cambridge.org/dictionary/english/selling) [goods](https://dictionary.cambridge.org/dictionary/english/goods) and [services](https://dictionary.cambridge.org/dictionary/english/service) of their country continuously without interruption".

Historic-Timeline-Outcome-Messages-Look:
If the player lose a war to a country, the game and their timeline ends in: ["PLAYER LOST THE WAR" ](https://i.postimg.cc/8cps30s1/Gamified-Compass-Player-Lost-War-Timelime-Stops2022-12-11-133658.png)
![GamifiedCompassPlayerLostWarTimelimeStops2022-12-11 133658](https://user-images.githubusercontent.com/118663590/206904905-37d61785-148e-41ea-a84a-e65a1875b334.png)


In https://i.postimg.cc/2Sv9ccWb/Government-Overthrown2022-12-11-133223.png and just one guy overthrowing it in this [Gam-Compliterallyjustoneguyoverthrow-The-Government2022-12-11-134743.png](https://postimg.cc/68SVpxSx), the timeline stops which the game signals that message with a sad, red smile and a thumbs-down-symbol. 
Successfully fought and lost wars are counted in [Assessment](https://i.postimg.cc/L6Fxqkwm/Gamified-Compass-Assessment2022-12-11-133444.png) with green being commonly positive in the "West" and red negative avoidance to loses. 
![GamifiedCompassAssessment2022-12-11 133444](https://user-images.githubusercontent.com/118663590/206904926-b8f851f1-a9a1-4194-aa94-eae852ffba43.png)
The message on the left says "You got conquered".

https://i.postimg.cc/wvHSPHWZ/Gam-Pol-Comp-Nuke-County-too-close-2022-12-11-155534.png and https://i.postimg.cc/fRcCPd6W/Gam-Pol-Comp-Surrender2022-12-11-154333.png , the message in the left says nothing, mainly because only the player themself is involved at this particular event.

In https://i.postimg.cc/HxhFL5WD/Gam-Pol-Comp-World-Peace-Geopolitical-Stability2022-12-11-153352.png , the message writer uses "!" in "World Peace!" https://i.postimg.cc/HxhFL5WD/Gam-Pol-Comp-World-Peace-Geopolitical-Stability2022-12-11-153352.png
![GamPolComp-World-Peace!-Geopolitical-Stability2022-12-11 153352](https://user-images.githubusercontent.com/118663590/206911581-78a38535-cb32-433e-9412-2502866aaec8.png)

**Comment by ilovepolitics** [comment]
Can you make a version for phone users?

**Reply by Fengsemul** [reply]
https://i.postimg.cc/ZqHnrBkD/Dev-Notes-2022-12-11-GMT-104026.png

> DEV NOTES / Direction of Project:
> 
> - No, unfortunately a phone/tablet version is not planned due to it being too far off the game plan (for now).

Source: https://canpixel.com/BadOptics/
